lecli/usage/commands.py

```diff
@@ -0,0 +1,18 @@
+"""
+Module for usage commands
+"""
+import click
+
+from lecli.usage import usage_api
```

Comment:
usage_api file can be renamed to only api imho. what do you think? because its already in a module called "usage".